README.md

```diff
@@ -81,16 +81,16 @@ For minimum requirements information and detailed instructions on how to use the
-| View test report (Gradle)            | Open the test report file, if it exists at the `build/reports/tests/test/index.html` default location. This action command is available only to Gradle projects. Gradle projects have only a single action command for test result reporting.                                                                                                                                                                             |
+| Start in a container                 | Start dev mode with Liberty running in a container. The `liberty-maven-plugin` must be version `3.3-M1` or higher. The `liberty-gradle-plugin` must be version `3.1-M1` or higher.                                                                                                                                                                                                                                       
+| Stop                                 | Stop dev mode. Liberty must be running in dev mode to use this command.                                                                                                                                                                                                                                                                                                                                                  |
+| Run tests                            | Run the unit tests and integration tests that are configured for your project. Liberty must be running in dev mode to use this command.                                                                                                                                                                                                                                                                                  |
+| View integration test report (Maven) | View the integration test report file if it exists at `target/site/failsafe-report.html` if Liberty dev mode used report plugin v3.4 or earlier or `target/reports/failsafe.html` otherwise.                                                                                                                                                                                                                             |
+| View unit test report (Maven)        | View the unit test report file if it exists at `target/site/surefire-report.html`  if Liberty dev mode used report plugin v3.4 or earlier or `target/reports/surefire.html` otherwise.                                                                                                                                                                                                                                   |
```

Comment:
Going forward, the new locations will become the more likely place to find the test reports. What's here is technically fine but would have probably emphasized the new location over the old one, rather than this word ordering.

Reply:
Ok